raspicam_node:
  framerate: 30
  enable_raw: false
  enable_imv: false